Sources/Examples/ConformanceMismatches.swift

```diff
@@ -0,0 +1,144 @@
+import Library
+
+// MARK: Under-Specified Protocol
+
+#if swift(<6.0)
+/// A conforming type that has now adopted global isolation.
+@MainActor
+class WindowStyler: Styler {
+	// Swift 5 Warning: main actor-isolated instance method 'applyStyle()' cannot be used to satisfy nonisolated protocol requirement
+	// Swift 6 Error: main actor-isolated instance method 'applyStyle()' cannot be used to satisfy nonisolated protocol requirement
+	func applyStyle() {
+	}
+}
+#endif
+
+// MARK: Globally-Isolated Protocol
+
+/// A conforming type that will infer the protocol's global isolation.
+class GloballyIsolatedWindowStyler: GloballyIsolatedStyler {
+	func applyStyle() {
+	}
+}
+
+/// A conforming type that will infer the protocol's global isolation *only*
```

Comment:
These feel like they need to explain more; without knowing full context I wouldn't know why from looking at this comment.

```suggestion
/// A type conforming to `PerRequirementIsolatedStyler` which has MainActor isolated protocol requirements,
/// will infer the protocol's requirements isolation for methods witnessing those protocol requirements *only*
```